.ruby-gemset

```diff
@@ -1 +1 @@
-registry
+rest-whois
```

Comment:
According to @teadur it has no bearing on production since servers are physically and logically separated. 

However, locally it produces an undesired behaviour inside `rvm`/`rbenv`: both `registry` and and `rest-whois` used the same gemset, which for example contained 2 different versions of rails (`4.2.10` and `4.2.7`). It also allowed gems to "sneak" from one environment to another:

<img width="945" alt="screen shot 2018-05-19 at 10 38 38" src="https://user-images.githubusercontent.com/2331765/40266276-d12b96ae-5b50-11e8-986f-f6534efadd6c.png">